src/components/ColumnAnnotationCard.tsx

```diff
@@ -138,12 +102,10 @@ function ColumnAnnotationCard({
             </Typography>
             <Autocomplete
               data-cy={`${id}-column-annotation-card-standardized-variable-dropdown`}
-              value={standardizedVariable?.label || ''}
-              onChange={handleStandardizedVariableChange}
-              options={Object.values(standardizedVariableOptions).map((value) => value.label)}
-              getOptionDisabled={(option) =>
-                mappedSingleColumnStandardizedVariables.map((sVar) => sVar.label).includes(option)
-              }
+              value={standardizedVariableLabel || ''}
+              onChange={(_, newValue) => onStandardizedVariableChange(id, newValue)}
+              options={standardizedVariableOptions}
+              getOptionDisabled={(option) => disabledStandardizedVariableLabels.has(option)}
```

Comment:
from docs: https://mui.com/material-ui/api/autocomplete/#autocomplete-prop-getOptionDisabled `option` will have same shape as an individual item. so if `options` is an array of objects rather than strings, then here you will be able to filter for the "enabled" status of these items instead of looking at labels